how-to-guides/e2e-tracking/notebooks/e2e_tracking.ipynb

```diff
@@ -0,0 +1,749 @@
+    "\n",
+    "This example uses Optuna hyperparameter-optimization to simulate training and evaluating multiple scikit-learn models, and Evidently to monitor models in production. However, given Neptune's flexibility and [multiple integrations](https://docs.neptune.ai/integrations/), you can use any library and framework of your choice. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Before you start\n",
+    "\n",
+    "This notebook example lets you try out Neptune anonymously, with zero setup.\n",
+    "\n",
+    "If you want to see the example logged to your own workspace instead:\n",
+    "\n",
+    "  1. Create a Neptune account. [Register &rarr;](https://neptune.ai/register)\n",
+    "  1. Create a Neptune project that you will use for tracking metadata. For instructions, see [Creating a project](https://docs.neptune.ai/setup/creating_project) in the Neptune docs."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Install Neptune and dependencies"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%pip install -q -U neptune[sklearn,optuna] scikit-learn optuna evidently matplotlib\n",
+    "%pip install -q --user scikit-learn matplotlib"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Run this if you get a `RuntimeError: main thread is not in main loop` error\n",
+    "import matplotlib\n",
+    "\n",
+    "matplotlib.use(\"Agg\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Track model training\n",
+    "\n",
+    "In this section, you will\n",
+    "1. Use Optuna to train multiple scikit-learn regression models\n",
+    "2. Leverage Neptune's [scikit-learn](https://docs.neptune.ai/integrations/sklearn/) and [Optuna](https://docs.neptune.ai/integrations/optuna/) integrations to automatically log metadata and metrics to Neptune for easy run comparison, while also using Neptune's [model registry](https://docs.neptune.ai/model_registry/overview/) to track models."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Prepare the dataset"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sklearn.model_selection import train_test_split\n",
+    "from sklearn.datasets import fetch_california_housing\n",
+    "\n",
+    "data, target = fetch_california_housing(return_X_y=True)\n",
+    "X_train, X_test, y_train, y_test = train_test_split(data, target, test_size=0.25)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create or initialize a Neptune project\n",
+    "This can also be done from the UI 👉 https://docs.neptune.ai/setup/creating_project/\n",
```

Comment:
Seems unnecessary, especially since we mention project creation in _Before you start_.